packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -623,7 +738,27 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     })
     this.storageService.on(StorageEvents.COMMUNITY_METADATA_STORED, async (meta: CommunityMetadata) => {
       this.logger(`Storage - ${StorageEvents.COMMUNITY_METADATA_STORED}: ${meta}`)
-      this.serverIoProvider.io.emit(SocketActionTypes.COMMUNITY_METADATA_STORED, meta)
+      const community = await this.localDbService.getCurrentCommunity()
+
+      if (community) {
+        const rootCaCert = loadCertificate(meta.rootCa)
+        const communityName = getCertFieldValue(rootCaCert, CertFieldsTypes.commonName)
+
+        if (!communityName) {
+          this.logger.error(`Could not retrieve ${CertFieldsTypes.commonName} from CommunityMetadata.rootCa`)
+        }
+
+        const updatedCommunity = {
+          ...community,
+          name: communityName ?? undefined,
+          rootCa: meta.rootCa,
+          ownerCertificate: meta.ownerCertificate,
+          ownerOrbitDbIdentity: meta.ownerOrbitDbIdentity,
+        }
+        await this.localDbService.setCommunity(updatedCommunity)
+
+        this.serverIoProvider.io.emit(SocketActionTypes.COMMUNITY_UPDATED, updatedCommunity)
+      }
```

Comment:
When another peer receives the CommunityMetadata from OrbitDB, they update their Community object in LevelDB and then notify the frontend via COMMUNITY_UPDATED.